src/marshmallow_sqlalchemy/convert.py

```diff
@@ -170,13 +170,7 @@ def field_for(self, model, property_name, **kwargs):
         prop = model.__mapper__.get_property(property_name)
         return self.property2field(prop, **kwargs)
 
-    def _get_key_for_field(self, prop_or_column):
-        """Hook to allow overriding property or column keys from input
-        SQLAlchemy model or table object. For simplicity, the same function
-        is used for both the `_fields_for_table` and `_fields_for_model`
-        methods, so the provided function should work for both columns
-        and ORM properties.
-        """
+    def _get_field_name(self, prop_or_column):
```

Comment:
attn @davenquinn : I think this is a more appropriate name. I'm also undocumenting this because I'm not quite comfortable with this being public API (yet).